packages/webpack/src/buildtime/emitSes.js

```diff
@@ -5,30 +5,28 @@ const {
 } = require('webpack')
 
 const lockdownSource = readFileSync(
-  path.join(require.resolve('ses'), '../lockdown.umd.min.js'),
+  path.join(require.resolve('ses'), '../lockdown.umd.js'),
   'utf-8'
 )
-const lockdownSourcePrefix = `;(function(){\n${lockdownSource}\n})();\n`
+const lockdownSourcePrefix = `/* SES sources included by LavaMoat. Do not optimize or minify. */\n;\n${lockdownSource}\n;/* end SES */\n`
```

Comment:
```suggestion
const lockdownSourcePrefix = `/*! SES sources included by LavaMoat. Do not optimize or minify. */\n;\n${lockdownSource}\n;/*! end SES */\n`
```

Reply:
Also: does adding inlining this befoul source maps?

Reply:
I'm afraid it does. But I'm also pretty sure wrapping each module does too. Might have to give it a separate push one day.

Reply:
Why did we give up on the anon func? Wasn't it important to avoid polluting upper scope by accident?

Reply:
SES is only polluting global scope in ways that are desired. There's plenty of wrapping involved.